_posts/en/2018-12-28-optech-review-2018-plan-2019.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Optech review of 2018 and plans for 2019'
+permalink: /en/optech-review-2018-plan-2019
+name: 2018-12-28-optech-review-2018-plan-2019
+type: posts
+layout: post
+lang: en
+version: 1
+---
+This month, Steve, Mike, James and I met to review the progress that Optech made
+in 2018 and make plans for 2019.
+
+We started Optech earlier this year because we wanted to [help companies adopt
+scaling technologies and raise the level of conversation around scaling
+Bitcoin][announcement]. This year, we’ve:
+
+- Signed up 15 companies as members, including exchanges, custodians, wallets and merchants.
+- Held [two workshops][workshops] to discuss scaling techniques with engineers
+  from member companies. Almost all attendees thought the workshops were ‘good’
+  or ‘excellent’, with an average score of 3.7/5 for the first and 4.2/5 for the
+  second.
+- Produced 1 pilot and 26 regular
+  [newsletters][]. If all the newsletters to date were printed out on typical
+  book pages, they'd be almost 100 pages long (33,000 words at 350 words per
+  page). Over 1700 Bitcoin engineers and enthusiasts have subscribed to the
+  newsletter.
+- Built a [dashboard][] to track the state of the Bitcoin network and the
+  adoption rate of scaling technologies. Marcin wrote about the [building the
+  dashboard in an Optech blog post][dashboard blog post]. 
```

Comment:
Very minor nit: Some trailing spaces here. I don't know if we care about that :-)